allure-web/build.gradle

```diff
@@ -5,12 +5,25 @@ apply from: "${gradleScriptDir}/bintray.gradle"
 apply plugin: 'maven'
 
 node {
-    version = '6.9.1'
+    version = '6.3.1'
     npmVersion = '3.10.3'
     download = true
 }
 
-processResources.dependsOn npm_run_build
+task buildWeb(type: NpmTask) {
+    inputs.files(fileTree('node_modules'))
+    inputs.files(fileTree('src'))
+    inputs.file('package.json')
+    inputs.file('jasmine.json')
+    inputs.file('webpack.config.js')
```

Comment:
What does this file list do?
Should we also add `.eslintrc` and `.babelrc` files there? Build will not work without them.